hw2/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,85 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args) {
+//   Задача 1.
+        var dog = 8.0;
+        var cat = 3.6;
+        var paper = 763789;
+
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+/*   Задача 2. Увеличьте значение каждой
+перечисленной в прошлой задаче переменной на 4.
+Для решения этой задачи используйте тип переменной var.
+ */
+        dog = 8.0 + 4;
+        cat = 3.6  + 4;
+        paper = 763789 + 4;
```

Comment:
```suggestion
        paper = paper + 4;
```
надо использовать переменную, а не число